packages/plugin-aws-lambda/src/lambda-timeout-approaching.js

```diff
@@ -0,0 +1,11 @@
+module.exports = class LambdaTimeoutApproaching extends Error {
+  constructor (remainingMs) {
+    const message = `Lambda will timeout in ${remainingMs}ms`
+
+    super(message)
+
+    this.name = 'LambdaTimeoutApproaching'
+    this.message = message
```

Comment:
is this line necessary? Calling `super(message)` should take care of setting it

Reply:
Yeah it should be unnecessary — it's here because originally I wasn't calling `super` but I forgot you *have* to call `super` in JS to use `this`